### PR TITLE
Parametrize test persistence

### PR DESCRIPTION
**Pull request by samgermain**
Switches the tests in test_persistence to be parametrized

### CODE REVIEW

File: freqtrade/utils/get_sides.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from typing import Tuple
+
+
+def get_sides(is_short: bool) -> Tuple[str, str]:
+    return ("sell", "buy") if is_short else ("buy", "sell")
```
Comment (xmatthias) [inline freqtrade/utils/get_sides.py]: I'm sceptical of generic "utils" packages/modules/... (or anything named similarily) ... as it'll end up as the "trashcan" for everything that can't go anywhere (i've been looking for that too very often, but usually finding a "properly named" place results in a better solution - or it'll end up in misc.py).

is your intend to use this also outside of tests in the future, or is it limited to tests?
If it's limited to tests (and has no benefit to the remainder of freqtrade) - then maybe moving it into the tests directory would make sense...

File: tests/test_persistence.py [hunk 2]
```diff
@@ -709,7 +590,7 @@ def test_trade_close(limit_buy_order_usdt, limit_sell_order_usdt, fee):
     assert trade.close_date == new_date
 
 
-@pytest.mark.usefixtures("init_persistence")
+@ pytest.mark.usefixtures("init_persistence")
```
Comment (xmatthias) [inline tests/test_persistence.py]: I'd not do that (and i'm not sure which stylesheet you've found this).

It looks odd - and is pretty dangerous when trying to find things - as it derives from the general style we're using elsewhere for decorators.

**Review by xmatthias — COMMENTED** [review]
it's nitpicks, really ... that's parametrization at it's best i'd say :+1:

File: tests/test_persistence.py (hunk 2 above)
Comment (samgermain) [reply tests/test_persistence.py]: Sorry, it's some auto-formatting thing that only happens sometimes, and  I haven't figured out what makes it happen

File: freqtrade/utils/get_sides.py (hunk 1 above)
Comment (samgermain) [reply freqtrade/utils/get_sides.py]: I moved it to conftest

File: tests/test_persistence.py (hunk 2 above)
Comment (xmatthias) [reply tests/test_persistence.py]: It's pretty strange, too - as it's not applied consistently (there's a ton of `@pytest` occurances in this same file which are not formatted this way).

**Review by xmatthias — APPROVED** [review]
LGTM :+1:

**Pull request merged**